src/client/eppo-precomputed-client.spec.ts

```diff
@@ -705,4 +705,51 @@ describe('EppoPrecomputedClient E2E test', () => {
 
     expect(loggedEvent.format).toEqual(FormatEnum.PRECOMPUTED);
   });
+
+  describe('EppoPrecomputedClient subject data and store initialization', () => {
```

Comment:
These tests show that we can use the subject data set by `setSubjectDataAndPrecomputedFlagStore` in our logs, vs needing `precomputedFlagsRequestParameters` which wouldn't be provided in an offline init